test/integration/resultsPage.js

```diff
@@ -25,7 +25,7 @@ function assertSearchResponse(search, postcode, done, assertions) {
 describe('Results page', () => {
   const noOnlineBookingLinkMessage = 'This surgery doesn&apos;t have an online booking system.';
 
-  describe('page layout', () => {
+  describe('layout', () => {
```

Comment:
Maybe an integration test for the presence of a greater than 0 resultLimit and resultsCount.